lib/data/demos.dart

```diff
@@ -201,6 +202,22 @@ List<GalleryDemo> materialDemos(GalleryLocalizations localizations) {
       ],
       category: GalleryDemoCategory.material,
     ),
+    GalleryDemo(
```

Comment:
This demo should be ordered alphabetically based on its title. Also I think the title should be updated to just `App bar` to match the widget itself. And then put it alphabetically.